src/Sylius/Bundle/CoreBundle/OAuth/UserProvider.php

```diff
@@ -62,7 +62,7 @@ public function __construct(
         RepositoryInterface $oauthRepository,
         ObjectManager $userManager,
         CanonicalizerInterface $canonicalizer,
-        CustomerRepositoryInterface $customerRepository
+        CustomerProviderInterface $customerProvider
```

Comment:
Unfortunately, changing the type of argument is a BC break :/ We should allow using both of them and introduce some check, together with deprecation info trigger

Reply:
@Zales0123 will this be fixed in 1.9? good time to introduce BC break :)

Reply:
@kayue we can't merge this solution in this way.
We Shouldn't couple API bundle with core bundle

Reply:
@AdamKasp I am aware. Feel free to fix this in a future release, just reporting bugs here. :)